<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YSAnalMessageBox_Dialog</class>
 <widget class="QDialog" name="YSAnalMessageBox_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analysis is running, please wait patiently ...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe UI&quot;;
background-color: rgb(43, 43, 43);

QPushButton::hover{background-color:rgb(144, 200, 246)}



</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,14,0,0" columnstretch="1,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QTextBrowser" name="YSMessageBox_textBrowser">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	background: rgb(43, 43, 43);
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="STime_label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Start Timeï¼š</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <property name="indent">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="StartTimeDay_label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="StartTimer_label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>112</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Stop_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover{background-color:rgb(144, 200, 246)}
QPushButton::disabled{color:rgb(153, 153, 153)}
QPushButton{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OK_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover{background-color:rgb(144, 200, 246)}
QPushButton::disabled{color:rgb(153, 153, 153)}
QPushButton{	
	font: 9pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 128);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="ExportMessage_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="styleSheet">
         <string notr="true">*{font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 255);
}</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Export message to:</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <property name="indent">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ExportFileFormat_comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>Text File Format</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="SaveText_toolButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(128, 128, 128);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
